CovertInfiltration/Src/CovertInfiltration/Classes/UIAvengerHUD_CAButton.uc

```diff
@@ -17,15 +17,13 @@ event OnInit(UIScreen Screen)
 	local UIStrategyMap StrategyMap;
 
 	StrategyMap = UIStrategyMap(Screen);
-	if (StrategyMap == none) return;
+	if(StrategyMap == none || `HQPRES.StrategyMap2D == none || `HQPRES.StrategyMap2D.m_eUIState == eSMS_Flight) return;

```

Comment:
I don't understand this change. Skip spawning any button *at all* when the map is currently in flight mode? When can this happen?

Reply:
Oh yeah, that was a failed attempt at fixing the "CA button shows up when avenger is in flight" issue. I forgot to remove it and it got pushed to this branch by accident.